frontend/packages/knative-plugin/src/components/overview/PodsOverview.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Link } from 'react-router-dom';
+import { K8sResourceKind, PodKind, podPhase } from '@console/internal/module/k8s';
+import { SidebarSectionHeading } from '@console/internal/components/utils';
+import {
+  PodsOverviewList,
+  podCompare,
+  isPodError,
+} from '@console/internal/components/overview/pods-overview';
+
+const kind: string = 'Pod';
+const MAX_PODS: number = 3;
+const MAX_ERROR_PODS: number = 10;
+
+type PodsOverviewProps = {
+  pods: PodKind[];
+  obj: K8sResourceKind;
+  text: string;
+};
+
+const PodsOverview: React.FC<PodsOverviewProps> = ({ pods, obj, text }) => {
```

Comment:
Very similar to `public/components/overview/pods-overview`, could that component be enhanced to provide the differences?

Reply:
yes if we can pass a  flag for link to `View All` and empty text message as props

Reply:
@jeff-phillips-18 have updated with extending `pods-overview`